rotary/rotary.py

```diff
@@ -112,16 +114,17 @@ def init(args):
 
     :param args: The command-line arguments.
     """
+    config = load_yaml_config(get_config_path(args, default_config_path=provided_config_path))
     output_dir_path = get_cli_arg_path(args, 'output_dir')
 
     # Checks for existing run files and sets up the run directory.
-    setup_run_directory(args, output_dir_path, run_files)
+    setup_run_directory(args, output_dir_path, run_files, config)
 
     input_path = get_cli_arg_path(args, 'input_dir')
 
-    dataset = generate_dataset_from_fastq_directory(input_path)
+    dataset = generate_dataset_from_fastq_directory(input_path, expected_files_per_sample=3)
```

Comment:
Just adding a note: we'll need to generalize this eventually when adding compatibility for long read-only samples.

Reply:
Resolved by: https://github.com/rotary-genomics/rotary/pull/146

Reply:
Great!